gbif-occurrence-linkage/main.py

```diff
@@ -121,14 +89,20 @@ def run_api_call(specimen_data: Dict) -> Dict[str, str]:
     response = requests.get(query_string)
     response_json = json.loads(response.content)
     if response_json['count'] == 1:
-        logging.info('Successfully retrieved a single result from GBIF based on the identifiers')
-        return {'queryString': query_string, 'gbifId': response_json['results'][0]['gbifID']}
+        logging.info(
+            'Successfully retrieved a single result from GBIF based on the identifiers')
+        return {'queryString': query_string,
+                'gbifId': response_json['results'][0]['gbifID']}
```

Comment:
Here you do use gbifID, it is probably good to go over all the usages of any ID and capitalize them